qtconsole/qtconsoleapp.py

```diff
@@ -205,6 +206,12 @@ def new_frontend_slave(self, current_widget):
         )
         kernel_client.load_connection_file()
         kernel_client.start_channels()
+        blocking_client = BlockingKernelClient(
+            connection_file=current_widget.kernel_client.connection_file,
+            config = self.config,
+        )
+        blocking_client.load_connection_file()
+        blocking_client.shell_channel.start()
```

Comment:
There's already a property to generate `blocking_client` - why is it being instantiated here? Won't it fail when you try to assign the attribute below, because the property is read only?